### ISSUE TITLE
Logux received wrong-subprotocol error but test app does not sent any request

### ISSUE THREAD

**Issue by egregors** [logux/backend-test#3]
When I try to execute `@logux/backend-test` I got `LoguxError` but the backend server actually does not receive any requests. 
```python
lbt git:(next) ✗ npx @logux/backend-test http://127.0.0.1:8000/logux/
Protocol version: 4
Secret:           parole
Logux server:     http://localhost:31337/
Back-end server:  http://127.0.0.1:8000/logux/

✖  0 Supports token authentication

Re-run test: npx @logux/backend-test http://127.0.0.1:8000/logux/ 0

LoguxError: Logux received wrong-subprotocol error (Only 0.0.0 application subprotocols are supported, but you use 1.0.0)
    at ClientNode.syncError (/Users/egregors/Documents/GitHub/django/lbt/node_modules/@logux/core/base-node/index.cjs:220:15)
    at ClientNode.errorMessage (/Users/egregors/Documents/GitHub/django/lbt/node_modules/@logux/core/error/index.cjs:10:8)
    at ClientNode.onMessage (/Users/egregors/Documents/GitHub/django/lbt/node_modules/@logux/core/base-node/index.cjs:193:27)
    at /Users/egregors/Documents/GitHub/django/lbt/node_modules/@logux/core/base-node/index.cjs:88:14
    at Object.emit (/Users/egregors/Documents/GitHub/django/lbt/node_modules/nanoevents/index.cjs:5:7)
    at Timeout._onTimeout (/Users/egregors/Documents/GitHub/django/lbt/node_modules/@logux/core/local-pair/index.cjs:59:30)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
```

**Comment by ai** [comment]
Fixed f848de4

Back-end Tests use `TestServer`, which is created for server unit tests. To make it easy to write tests I added default `supports` and `auth` options.

Now Back-end Tests will be closer to reality.

**Comment by ai** [comment]
Th fix was released in 4.0.3
